Topics/domain-registration.md

```diff
@@ -0,0 +1,9 @@
+# How to register a domain name
+
+## What are domains?
+
+Websites are reached over the interted using IP addresses, however IP addresses are not very convenient for users as they are hard to remember. Domains are names that are assigned to IP addresses for ease of access. For example instead of accesing a website by an IP of 192.158.1.38 you can access it by domainname.com.
+
+## How do I register a domain for my website?
+
+In order to attach a domain do your webserver you will need to purchase a domain from a Domain Name Registar such as GoDaddy or Namecheap. Prices will vary based on how popular the domain name is. After you have purchased the domain you then have to connect it to the server hosting your website. Most Domain Registars will have a built in system for connecting domain names to servers that you will have to use to connect the two together. After which you can then use your new domain name to connect to your server. 
```

Comment:
I like how straight-to-the-point your tutorial is!

The only suggestion is to have a section where you further elaborate on questions that your reader may have, e.g.
- How does an IP address get mapped to a domain name?
- How do I tell my server about my new domain name? (e.g. how we set it up for our application using AWS)

I'd also say include a section where the reader can be pointed to for further information.

Reply:
added